src/webportal/src/app/job/job-view/fabric/job-detail/conn.js

```diff
@@ -90,8 +90,34 @@ export async function fetchJobRetries() {
   }
 }
 
-export async function fetchJobInfo() {
-  return wrapper(() => client.job.getJob(userName, jobName));
+export async function fetchJobInfo(attemptIndex) {
+  return wrapper(async () => {
+    const restServerUri = new URL(config.restServerUri, window.location.href);
+    const url = isNil(attemptIndex)
+      ? `${restServerUri}/api/v2/jobs/${userName}~${jobName}`
+      : `${restServerUri}/api/v2/jobs/${userName}~${jobName}/attempts/${attemptIndex}`;
+    const res = await fetch(url, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    const result = await res.json();
+    return result;
+  });
+}
+
+export async function fetchTaskStatus(attemptIndex, taskRoleName, taskIndex) {
+  return wrapper(async () => {
+    const restServerUri = new URL(config.restServerUri, window.location.href);
+    const url = `${restServerUri}/api/v2/jobs/${userName}~${jobName}/attempts/${attemptIndex}/taskRoles/${taskRoleName}/taskIndex/${taskIndex}/attempts`;
+    const res = await fetch(url, {
```

Comment:
Align with RestServer, the WebUI also use below URL?
/jobs/${userName}~${jobName}/attempts/${attemptIndex}/taskRoles/${taskRoleName}/taskIndex/${taskIndex}/attempts`;